### DISCUSSION TITLE
alist设置带宽限制

### DISCUSSION THREAD

**Discussion by Hemingway2003** [AlistGo/alist#6019] [Q&A] [answered]
请问如何设置alist的带宽限制呢，目前服务跑在购买的服务器上，上传带宽有限，如果设置将本地内容复制到云盘的话，会打满该服务器的带宽，导致复制未完成期间无法登录或对服务器进行其他操作
感谢

**Comment by Hemingway2003** [comment]
用的是直接装，并非是docker的，因为docker的不好确定用户id

**Comment by xhofe** [comment]
https://www.tecmint.com/limit-linux-network-bandwidth-usage-with-trickle/

**Reply by Hemingway2003** [reply]
感谢答复，但是trickle应该只针对用了lib6动态库的程序有用，而alist是一个statically linked程序，所以trickle是无效的

**Reply by xhofe** [reply, marked as answer]
emmm 那我不知道有没有别的方法了 初始设计的时候没有考虑到这方面，要从alist自身支持的话可能需要整体的重构。

**Reply by Hemingway2003** [reply]
再次感谢，那只能通过创建虚拟网卡，限制该网卡速率，并将alist改到对应网卡上的方法了

**Reply by hankqqu** [reply]
> emmm 那我不知道有没有别的方法了 初始设计的时候没有考虑到这方面，要从alist自身支持的话可能需要整体的重构。

**Comment by huolanyun** [comment]
Alist 载体设备限速，例如linux ，针对IP MAC 路由器都可以限速
